### PR TITLE
fix: missing compiled (.mo files) translations

### PR DESCRIPTION
**Pull request by fabien-michel**
Restore compiled translations (.mo) files in repository

Should fix: https://github.com/Pierre-Sassoulas/django-zxcvbn-password-validator/issues/18

### CODE REVIEW

File: django_zxcvbn_password_validator/zxcvbn_password_validator.py [hunk 1]
```diff
@@ -87,9 +87,9 @@ def append_translated_feedback(old_feedbacks, feedback_type, new_feedbacks):
     def get_help_text(self):
         expectations = _("We expect")
         if self.password_minimal_strength == 0:
-            expectations += f" {_('nothing: you can use any password you want.')}"
+            expectations += " " + _('nothing: you can use any password you want.')
             return expectations
-        expectations += f" {_('a password that cannot be guessed')}"
+        expectations += " " + _('a password that cannot be guessed')
```
Comment (Pierre-Sassoulas) [inline django_zxcvbn_password_validator/zxcvbn_password_validator.py:92]: Why is this part modified ?

Comment (fabien-michel) [reply django_zxcvbn_password_validator/zxcvbn_password_validator.py:92]: 🤷‍♂️ Don't know why but `makemessages` doesn't collect the translations here

Comment (fabien-michel) [reply django_zxcvbn_password_validator/zxcvbn_password_validator.py:92]: 🤷‍♂️ Don't know why but makemessages doesn't collect the translations here
So they are suppressed from the .po file. Maybe a recent modification in how gettext work ?

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
Look good, thank you. Could you launch pre-commit and fix the test locally ? There seems to be an issue right now with push right for pre-commit.

**Review by Pierre-Sassoulas — COMMENTED** [review]
Shouldn't there be a mo file added for the nl language too ? I would also expect an addition to the packaging so it include .mo file in the result of ``python setup.py build sdist`` (unless it's already working, sorry I'm not able to check that right now).

**Comment by fabien-michel** [comment]
The `.mo` file for `nl` is also added.
Sorry I do not use to create python packages. I think it should be ok, when running the command you give I can see the .mo files  in the `egg-info` folder

**Review by Pierre-Sassoulas — APPROVED** [review]
> Sorry I do not use to create python packages. I think it should be ok, when running the command you give I can see the .mo files in the egg-info folder

No problem, I'll check that before releasing 1.4.2, thank you for the contribution, much appreciated ! 😄
